### ISSUE TITLE
VS2022 Mlpack build error 

### ISSUE THREAD

**Issue by NBarile** [mlpack/mlpack#3184]
#### Issue description

Hi guys !
I Trying to build mlpack with VS2022, an i have somes errors when I build the solution...

`Erreur LNK2019	symbole externe non résolu "__declspec(dllimport) public: static class boost::unit_test::unit_test_log_t & __cdecl boost::unit_test::unit_test_log_t::instance(void)" (__imp_?instance@unit_test_log_t@unit_test@boost@@SAAEAV123@XZ) référencé dans la fonction "void __cdecl boost::unit_test::`anonymous namespace'::`dynamic initializer for 'unit_test_log''(void)" (??__Eunit_test_log@?A0xb724a72f@unit_test@boost@@YAXXZ)`

`Erreur	LNK2001	symbole externe non résolu "__declspec(dllimport) public: static class boost::unit_test::unit_test_log_t & __cdecl boost::unit_test::unit_test_log_t::instance(void)" (__imp_?instance@unit_test_log_t@unit_test@boost@@SAAEAV123@XZ)`

If someone can help me, I follow all the instruction here :  https://www.mlpack.org/doc/mlpack-git/doxygen/build_windows.html

Thanks !
If you need more informations, please ask me !

#### Your environment

 * version of mlpack: 3.4.2
 * operating system: win10
 * compiler: CMake
 * version of dependencies (Boost/Armadillo): Boost : boost_1_78_0-msvc-14.3-64 / Armadillo : 11.0.0

**Comment by NBarile** [comment]
oh, okay

And so, I just have to use the branch master, and follow the same instruction ?

**Comment by zoq** [comment]
Yes, or download the zip archive - https://github.com/mlpack/mlpack/archive/refs/heads/master.zip.

**Comment by NBarile** [comment]
Ok

Thanks a lot guys :D
